src/App.jsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from 'react'
+import './App.css'
+import Die from '../components/Die'
+import { nanoid } from 'nanoid'
+import Confetti from "react-confetti"
+
+
+function App() {
+  const [count, setCount] = useState(allNewDice())
```

Comment:
For better performance, you can use lazy state initialization
```suggestion
  const [count, setCount] = useState(() => allNewDice())
```